lib/rules/selector-max-specificity/README.md

```diff
@@ -79,3 +79,43 @@ div {}
   }
 }
 ```
+
+## Optional secondary options
+
+### `ignoreSelectors: ["/regex/", "string"]`
+
+Given:
+
+```js
+["0,2,0", {
+  ignoreSelectors: [":global", ":local", "/my-/"]
+}];
+```
+
+The following patterns are *not* considered violations:
+
+```css
+:global(.foo) .bar {}
+```
+
+```css
+:local(.foo.bar)
+```
+
+```css
+:local(.foo, :global(.bar).baz)
+```
+
+The following patterns are still considered violations:
```

Comment:
You weren't to know this, but this must read exactly:

```
The following patterns are considered violations:
```

For the red/green background color highlighting to work on [the website](https://stylelint.io/user-guide/rules/color-no-invalid-hex/).